docs/usage/self-hosted-configuration.md

```diff
@@ -7,6 +7,25 @@ description: Self-Hosted Configuration usable in renovate.json or package.json
 
 The below configuration options are applicable only if you are running your own instance ("bot") of Renovate.
 
+## allowPostUpgradeCommandTemplating
+
+If true allow templating for post-upgrade commands.
+
+e.g.
+
+```json
+{
+  "allowPostUpgradeCommandTemplating": true,
```

Comment:
This option would normally be configured in the `config.js` by the bot administrator. Also, `allowedpostupgradecommands` would normally be configured next to it, so good to include in the example.

Reply:
Yes, i think this should be two samples, one for bot config with `allowedPostUpgradeCommands`. and another for the `renovate.json` repo config.

Reply:
Make an example for angular/core with `allowedPostUpgradeCommands` and `postUpgradeTasks` fields